content/protocols/mqtt.textile

```diff
@@ -0,0 +1,150 @@
+---
+title: MQTT
+meta_description: "Any MQTT-enabled client can communication with the Ably service through the Ably MQTT protocol adapter. This is especially useful where an Ably client library SDK is not available for your language of choice."
+---
+
+The Ably MQTT protocol adapter is able to translate back and forth between "MQTT":https://mqtt.org/ and Ably's own protocol, allowing for seamless integration of any systems you may have. MQTT (MQ Telemetry Transport) is a "publish/subscribe":https://ably.com/topic/pub-sub , lightweight messaging protocol designed for constrained devices and low-bandwidth networks. One of the major uses of MQTT is with IoT (Internet of Things), where these principles are key to having effective communication between various devices.
+
+h2(#when-to-use). When to use the MQTT adapter
+
+The MQTT adapter is our recommended way of interacting with Ably from devices which do not have a native Ably client library, such as Arduino platforms, C/C++ applications, and so on. Anyone who has previously been using Pubnub or Pusher client libraries for this purpose may want to consider switching to MQTT. Compared to the Pubnub protocol, using MQTT will result in better performance with lower latency. Compared to the Pusher protocol, MQTT will give you connection state recovery.
+
+We recommend you use MQTT to interact with Ably when:
+
+* Using devices which do not have a device-specific Ably Client Library SDK, such as Arduino platforms, C/C++ applications, and so on
+* If bandwidth is limited and you want to keep network traffic to a minimum
+* You want to keep network traffic to a minimum.
+
+Note, we do not currently support MQTT over websockets but let us know if you are interested.
+
+h2(#limitations). Known limitations 
```

Comment:
This section needs to be reviewed for readability/clarity, e.g. "though presence may be supported soon".